src/game_battlealgorithm.cpp

```diff
@@ -1127,6 +1127,10 @@ const RPG::Sound* Game_BattleAlgorithm::Skill::GetStartSe() const {
 	}
 }
 
+const RPG::Sound* Game_BattleAlgorithm::Skill::GetResultSe() const {
+	return !success && skill.failure_message != 3 ? NULL : AlgorithmBase::GetResultSe();
```

Comment:
This magic number should probably be resolved to some `SkillFailureMessage_Physical` in the future.